status.py

```diff
@@ -150,9 +153,12 @@
             },
             "Off_Status": {
                 "tags": [ TAG.Off, TAG.Status ],
-            },
-            "On_Status": {
-                "tags": [ TAG.On, TAG.Status ],
+                "subclasses": {
+                    "Emergency_Power_Off_Status": {}, # defined elsewhere
```

Comment:
I'd rather use "parents" annotation to be consistent with others. Then, we can easily say it is wrong if the class has an empty definition.

Reply:
Good point. There's a few of these laying around from before I added the "parents" annotation. I fixed this one but I'll try to find the others as well